src/js/redux/reducers/award/awardReducer.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * awardReducer.js
+ * Created by Emily Gullo 01/23/2017
+ **/
+
+const initialState = [];
```

Comment:
As per conversation, altering initial state to be an object, rather than an array.